SDK-1204 Updated docs (#686)
box/box-ios-sdk: docs/usage/recent-items.md

----------------

To get recently accessed items, call

> [`client.recentItems.list(marker: String?, limit: Int?, fields: [String]?)`][get-recent-items]. This method will return an iterator object in the completion you can use to retrieve recent items.

Reviewer: ```suggestion
[`client.recentItems.list(marker: String?, limit: Int?, fields: [String]?)`][get-recent-items]. This method will return an iterator object in the completion, which is used to retrieve recent items.
```
